{"DKkj6IEFz0ArpaGh": "Traceback (most recent call last):\n  File \"ATRI/plugins\\plugin_chat\\__init__.py\", line 179, in _\n    await fxxkMe.finish(request_api_text(URL))\n  File \"C:\\Users\\Administrator.DESKTOP-D764LND\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\nonebot\\matcher.py\", line 583, in finish\n    raise FinishedException\nnonebot.exception.FinishedException\n", "piNxHF16v4mWODBX": "Traceback (most recent call last):\n  File \"ATRI/plugins\\plugin_utils\\__init__.py\", line 106, in _\n    uid_info = JsonAnalysis(GetInfo(uid))\n  File \"ATRI/plugins\\plugin_utils\\genshin.py\", line 66, in JsonAnalysis\n    Character_List = data[\"data\"][\"avatars\"]\nTypeError: 'NoneType' object is not subscriptable\n"}